webnode/src/components/consent-overlay/logo-group/index.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from "react";
+import Logo from "../../../assets/images/oyster-logo-with-dot.svg";
```

Comment:
Im not sure how this is working, you cannot import an image because it is not a module, instead you have to declare a constant and require() it

Reply:
I tried many ways but this is only one what works for me